### DISCUSSION TITLE
Transcode DTMF tones

### DISCUSSION THREAD

**Discussion by rsolalinde** [hnimminh/libresbc#157] [Q&A]
Hello, Is there a way to transcode DTMF RFC 2833 to DTMF inband?. For example peer A only talk with dtmf RFC 2833 and peer B only accept dtmf inband.

**Comment by hnimminh** [comment]
It should be done with media class, where you set dtmf mode (rfc2833, info)

https://github.com/hnimminh/libresbc/wiki/Media

**Reply by rsolalinde** [reply]
Hello, Thank you for your reply, but I want to transcode RFC 2833 to passthrough in-band, and in the media class section there isn't a passthrough in-band option.
